### PR TITLE
[dimcli/draco/refprop-headers/coolprop] Update version

### PR DESCRIPTION
**Pull request by JackBoosY**
Continue PR #13558.
Related: #13536.

### CODE REVIEW

**Review by LilyWangL — APPROVED** [review]

File: ports/refprop-headers/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: refprop-headers
-Version: 2017-11-7-882aec454b2bc3d5323b8691736ff09c288f4ed6
+Version: 2019-04-14-8df35c7cb557d385bb66431e92e836104a63a33c
```
Comment (ras0219-msft) [inline ports/refprop-headers/CONTROL:2]:
```suggestion
Version: 2019-04-14
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

**Comment by JackBoosY** [comment]
Blocked by #13722.

**Comment by JackBoosY** [comment]
blocked by #13765.

**Comment by ras0219-msft** [comment]
LGTM, thanks!

**Pull request merged**